### ISSUE TITLE
Unable to edit scripts in nightly x64

### ISSUE THREAD

**Issue by cyberrufus** [greasemonkey/greasemonkey#2073]
Inability to edit scripts in the latest nightly x64 (text editor fails to launch). Extension works as expected in FF 35 (x32). Using Greasemonkey 2.3.

**Comment by janekptacijarabaci** [comment]
Not confirmed on (OS Windows):
Firefox 38.0a1 x64 (2015-01-16) Nightly (Enable/Disable E10S)
Greasemonkey 2.3

**Comment by cyberrufus** [comment]
Should have noted my OS as Windows 8.1. Already tried disabling and enabling E10S and it had no effect.

**Comment by janekptacijarabaci** [comment]
Confirmed.

May I suggest this fix (Greasemonkey 2.3/3.0+ (e10s_sync)):
https://github.com/arantius/greasemonkey/blob/e10s_sync/content/scriptprefs.js#L3

From:
```javascript
var gScriptId = location.hash.substring(1);
```

To:
```javascript
var gScriptId = decodeURIComponent(location.hash.substring(1));
```

**Comment by trvrfrd** [comment]
Having the same issue in FF Developer Edition/Aurora 38.0a2 on OS X 10.10.2.

**Comment by stormdr1ve** [comment]
Same issue using FF developer edition/38.0a2 64bit on Windows 7.

**Comment by arantius** [comment]
This fix is now included in version 3.1beta1.  Please browse to [Greasemonkey's AMO page](https://github.com/Ventero/greasemonkey/commit/b8a09369482f87d0d9fa8afb39b5f46439eab0c2) and open "Development Channel" near the bottom to install the beta version.  Can you confirm the fix?  Thank you.

**Comment by Betsy25** [comment]
Problem is fixed on my side with 3.1beta1.
Thanks :+1:

**Comment by stormdr1ve** [comment]
Fixed for me with 3.1b1 in Firefox dev edition 38.0a2 x64 on Windows 7

**Comment by trvrfrd** [comment]
Fix confirmed for me as well, thank you! :+1:
